alkira/resource_alkira_peering_gateway_cxp.go

```diff
@@ -0,0 +1,173 @@
+package alkira
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/alkiranet/alkira-client-go/alkira"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceAlkiraPeeringGatewayCxp() *schema.Resource {
+	return &schema.Resource{
+		Description:   "Manage CXP Peering Gateways.",
+		CreateContext: resourceAlkiraPeeringGatewayCxpCreate,
+		ReadContext:   resourceAlkiraPeeringGatewayCxpRead,
+		UpdateContext: resourceAlkiraPeeringGatewayCxpUpdate,
+		DeleteContext: resourceAlkiraPeeringGatewayCxpDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Description: "The name of the Peering Gateway.\nOnce deployed this property can not be changed. All the changes will be ignored.",
```

Comment:
Try to keep all those description at 80 per line... so it will be much easier to manage.

Reply:
And "all changes will be ignored" is not a good behaviour... we should fix that and API should report errors back if someone tries to update this field.

Reply:
updated